build.gradle

```diff
@@ -17,7 +18,6 @@ buildscript {
 
 plugins {
   id "ru.vyarus.animalsniffer" version "1.5.3"
-  id "com.github.johnrengelman.shadow" version "5.2.0"
   id "me.champeau.gradle.japicmp" version "0.2.9"
   id "com.diffplug.spotless" version "5.1.2"
   id "net.ltgt.errorprone" version "1.2.1"
```

Comment:
And I never see the use of `spotless` and `errorprone` plugins now, if them could be removed ?

Reply:
Errorprone and spotless are activated by including the plugin if I remember correctly.

Reply:
Ok, I see the `spotlessCheck` task run with `./gradlew check -x test`.